app/src/main/java/com/pajato/android/gamechat/database/DBUtils.java

```diff
@@ -97,6 +97,9 @@ public static int getUnseenExperienceCount(@NonNull final String groupKey,
     public static int getUnseenMessageCount(@NonNull final String groupKey,
                                             @NonNull final Map<String, Integer> map) {
         int result = 0;
+        Map<String, Map<String,Message>> msgMap = MessageManager.instance.messageMap.get(groupKey);
```

Comment:
Something does not look right but it might be that not enough code is showing: msgMap does not seem to get used within the shown context.